keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -411,7 +405,7 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
-We manually test it in our internal environment and it works.
+We use the feature in our internal environment and it works.
```

Comment:
I'd expect a detailed description how the upgrade->downgrade->upgrade path was tested. The above description says you use the feature, but that doesn't answer the question about upgrade->downgrade->upgrade path.

Reply:
OK, it is really the same as feature activation and de-activation. I will change to explicit steps here.